karma.conf.js

```diff
@@ -123,6 +127,19 @@ module.exports = function(config) {
       });
     }
 
+    if (options['native-iterator']) {
+      customLaunchers = {
+        SL_Chrome: {
+          base: 'SauceLabs',
+          browserName: 'chrome'
+        },
+        SL_Firefox: {
+          base: 'SauceLabs',
+          browserName: 'firefox'
+        }
```

Comment:
I tried both doing it with gesalakacula and without gesalakacula. And when I tried it without, I started out using a specific "version" property but then removed the property because it didn't seem to work. See https://travis-ci.org/ModuleLoader/es6-module-loader/pull_requests for all of my attempts at getting this thing to pass :)

The reason why you can't see all of those commits is because I was squashing and force pushing, in an effort to keep this pr at a reasonable number of commits.